src/model.rs

```diff
@@ -1807,17 +1806,17 @@ mod tests {
             let mut collections = collections(3);
             collections.update_stop_area_coords();
             let stop_area = collections.stop_areas.get("stop_area:1").unwrap();
-            assert_eq!(2.0, stop_area.coord.lon);
-            assert_eq!(2.0, stop_area.coord.lat);
+            assert!((stop_area.coord.lon - 2.0).abs() < std::f64::EPSILON);
```

Comment:
Not using `assert_relative_eq!` on these one, any reason?

Reply:
bad find and replace by regex . Thx